test/EndToEnd/WorkloadTests.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Xml.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace EndToEnd.Tests
+{
+    public class WorkloadTests : TestBase
+    {
+        [Fact]
+        public void ItCannotPublishBlazorWasm_AOTWithoutWorkloadInstalled()
+        {
+            DirectoryInfo directory = TestAssets.CreateTestDirectory();
+            WorkloadTestEnvironment env = PrepareTestEnvironment(directory.FullName);
+
+            //FIXME: poke the value in the project file
```

Comment:
What does this mean?

Reply:
I want to set the property in the project file, instead of forcing it as a global property from the command line. That would be closest to what a user would do.